nextcord/file.py

```diff
@@ -95,6 +117,7 @@ def __init__(
         *,
         description: Optional[str] = None,
         spoiler: bool = False,
+        force_close: bool = True,  # Accessed in .close()
```

Comment:
I believe it should default to `True` since there are rarely any cases where you would need persistent bytes after using the file.
But I'm open to getting more opinions from others too.

Reply:
As the discussion you guys have had above says, keeping it to the original behaviour avoids breaking changes (user closing it manually may throw an error)

Reply:
> As the discussion you guys have had above says, keeping it to the original behaviour avoids breaking changes (user closing it manually may throw an error)

The concern isn't that. The user should be able to call `.close()` multiple times without a problem. That includes if the file is automatically closed. The concern lies only if the user plans to reuse the file bytes for whatever reason.
But with due respect, since you both think it's a better idea to keep `force_close` False for user-created files.